src/status_im/ui/screens/chat/components/input.cljs

```diff
@@ -564,18 +590,18 @@
          [:f>
           (fn []
             (let [reply (<sub [:chats/reply-message])
+                  suggestions (<sub [:chat/mention-suggestions])
                   {window-height :height} (rn/use-window-dimensions)
                   {:keys [keyboard-shown keyboard-height]} (rn/use-keyboard)
-
                   max-y (- window-height (if (> keyboard-height 0) keyboard-height 360) (:top insets)) ; 360 - default height
                   max-height (- max-y 56 (:bottom insets))  ; 56 - top-bar height
-                  added-value (if reply 38 0) ; increased height of input box needed when reply
-                  min-y (+ min-y added-value)
+                  added-value (if (and (not (seq suggestions)) reply) 38 0) ; increased height of input box needed when reply
+                  min-y (+ min-y (when reply 38))
                   y (calculate-y context keyboard-shown min-y max-y added-value)
```

Comment:
could we have all calculations with conditions in `calculate-y` ? so we have only one calculate-y function ? because we 'll have more options

Reply:
@flexsurfer `added-value` was already inside `calculate-y` in the reply component issue, but we moved outside to use it to update the input height.

Reply:
i mean move `calculate-y-with-mentions` code to `calculate-y` function, so we have only on

Reply:
@flexsurfer there are many calculations in both of them, and having both of them in one function maybe difficult to understand, so I created a function `get-y-value` that calls both of them.